diffpy/pdfmorph/tests/test_morphsequence.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+
+import unittest
+from pathlib import Path
+
+from diffpy.pdfmorph.pdfmorphapp import create_option_parser, single_morph, multiple_morphs
+
+thisfile = locals().get('__file__', 'file.py')
+tests_dir = Path(thisfile).parent.resolve()
+testsequence_dir = tests_dir.joinpath("testdata").joinpath("testsequence")
+
+
+class TestMorphSequence(unittest.TestCase):
```

Comment:
per my comment above, we are moving to using pytest because the syntax is easier and personally I never learned unittest syntax so I find it harder to maintain.  As per above, we won't go back and change old unittest code to pytest, but can we use pytest syntax in all new tests and older tests that we revisit for some reason?

Reply:
I'll figure out how to move setUp() or reconstruct it. Should not be too difficult to change